packaging/build_multiversion_debs

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
`build_multiversion_debs` sounds like the deb packages themselves are multi-versioned. I suggest `build_all_debs`.